README.developer.md

```diff
@@ -0,0 +1,18 @@
+# NMState Coding Guidelines
+
+- NMState is written primarily in Python, and its coding style should follow the best practices of Python coding unless otherwise declared.
+- PEP8 is holy.
```

Comment:
How about mentioning that commits should be checked with tox here, it includes a pep8 check afaik.